src/ocamlorg_frontend/pages/about.eml

```diff
@@ -50,243 +50,168 @@ Layout.render
-                        academia, OCaml has always put great emphasis on performance. It has developed complexity
-                        alongside powerful methods to manage that complexity. As a result, OCaml is uniquely poised to
-                        adapt to modern challenges whilst retaining a strong well-proven core.
-                    </p>
-                    <h3 id="strength">Strengths</h3>
                     <p>
-                        A language’s strong points are what define it, and OCaml’s strengths have made it popular with
-                        individuals and companies alike. The community that surrounds the language is one of its
-                        greatest assets, and a constant source of innovation.
+                       Programming languages matter. They affect the reliability, security, and efficiency of the code
+                       you write, as well as how easy it is to read, refactor, and extend. The languages you know can
+                       also change how you think, influencing the way you design software even when you’re not using
+                       them.
                     </p>
                     <p>
-                        On the technical side, as a functional language, OCaml places an emphasis on deterministic
-                        mathematical functions that limit side effects. Arguably this contributes to fewer bugs and
-                        easier formal verification, debugging, and testing. Furthermore, programs are verified by the
-                        compiler before they are executed, which eliminates a lot of human programming errors. This
-                        makes the language safer, a crucial trait in environments where a single mistake can have
-                        devastating consequences. Since OCaml is so safe it is currently used both commercially in the
-                        financial market, and academically as a teaching and research language in universities across
-                        the world.
+                       OCaml mixes power and pragmatism in a way that makes it ideal for building complex
+                       software systems. What makes OCaml special is that it occupies a sweet spot in programming language design. It provides a combination of efficiency, expressiveness, and
+                       practicality that is matched by no other language. That is in large part because OCaml is an
+                       elegant combination of language features that have been developed over the last 40
+                       years. These include:
                     </p>
-                    <h6 class="font-bold text-body-600">Why is OCaml a Good Choice? It has:</h6>
                     <ul>
                         <li>
-                            <strong>A strong community:</strong>
-                            The open-source community that surrounds OCaml is large and passionate. It is made up of
```

Comment:
How can we not mention the community here? It's a core part of what makes OCaml great